mlflow/server/auth/__init__.py

```diff
@@ -8,15 +8,60 @@
 """
 
 import logging
+import uuid
+import os
+from pathlib import Path
+from flask import Flask, request, make_response, Response, redirect, flash, render_template_string
 
 from mlflow.server import app
+from mlflow.server.auth.config import read_auth_config
+from mlflow.server.auth.sqlalchemy_store import SqlAlchemyStore
+from mlflow.server.handlers import _get_rest_path
 
+_AUTH_CONFIG_PATH_ENV_VAR = "MLFLOW_AUTH_CONFIG_PATH"
 
 _logger = logging.getLogger(__name__)
 
+store = SqlAlchemyStore()
+
+
+class ROUTES:
+    USERS = _get_rest_path("/mlflow/users")
+    SIGNUP = "/signup"
+
+
+UNPROTECTED_ROUTES = [ROUTES.USERS, ROUTES.SIGNUP]
+
+
+def is_unprotected_route(path: str) -> bool:
+    if path.startswith("/static") or path.startswith("/favicon.ico"):
```

Comment:
```suggestion
    if path.startswith(("/static", "/favicon.ico")):
```

`startswith` accepts a tuple :)


```python
def startswith_foo_or_bar(s: str):
    return s.startswith(("foo", "bar"))


print(startswith_foo_or_bar("foo"))  # True
print(startswith_foo_or_bar("bar"))  # also True

```

Reply:
great, I didn't know that!